x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/parser/LikeEscapingParsingTests.java

```diff
@@ -29,7 +29,7 @@ private String error(String pattern) {
     }
 
     private LikePattern like(String pattern) {
-        Expression exp = null;
+        Expression exp;
```

Comment:
Please revert.

Reply:
Will do. I was wondering what is your convention.

Reply:
Generally speaking, I don't have a strong preference for explicitly initialising as `null` but some times it helps for readability. In this case we have a clear if/else statement that assigns a value, so imho is not so important.